Modix.Bot/Behaviors/PromotionPollBehavior/CachedPromoPoll.cs

```diff
@@ -0,0 +1,17 @@
+using Discord;
+using Modix.Data.Models.Promotions;
+
+namespace Modix.Bot.Behaviors
+{
+    public class CachedPromoPoll
```

Comment:
Nitpick: Refactor `Poll` to `Dialog` or something whatever, to match the behavior name. Alternatively, don't even make a custom class here. You can use the existing `PromotionCampaignSummary` class out of `Modix.Data` and either re-build it manually to change the counts, or just re-query it on each update (to make sure you never display wrong data), and only cache the `MessageId`/`CampaignId` mappings.